linker/Mono.Linker/Annotations.cs

```diff
@@ -316,9 +316,10 @@ public void SaveDependencies ()
 			writer.WriteEndElement ();
 			writer.WriteEndDocument ();
 			writer.Flush ();
+#if !NET_CORE
```

Comment:
XmlWriter and GZipStream don't have Close methods on core: [4797](https://github.com/dotnet/corefx/issues/4797). Looking at this again, I think it would be safe to get rid of the calls to Close() altogether, as Dispose() should take care of it for us on desktop: [GZipStream](https://referencesource.microsoft.com/#System/sys/System/IO/compression/GZipStream.cs,18658ab48f2f7361), [XmlWriter](https://referencesource.microsoft.com/#System.Xml/System/Xml/Core/XmlWriter.cs,e5032b905808fabc). What do you think?